BoxContentSDK/BoxContentSDK/Requests/BOXFolderDeleteRequest.m

```diff
@@ -31,6 +41,21 @@ - (instancetype)initWithFolderID:(NSString *)folderID isTrashed:(BOOL)isTrashed
     return self;
 }
 
+- (instancetype)initWithFolderID:(NSString *)folderID
+                       isTrashed:(BOOL)isTrashed
+                     associateId:(NSString *)associateId
+{
```

Comment:
minor: re-use the current initWithFolderID:isTrashed: method, and add assignment for associateId

Reply:
done